crates/oxc_linter/src/lib.rs

```diff
@@ -23,6 +23,7 @@ use rustc_hash::FxHashMap;
 
 pub use crate::{
     context::LintContext,
+    fixer::Fix,
```

Comment:
We can expose this or we can add an `impl From<(String, Span)> for Fix`, but this is fine I think

Reply:
This is fine.